### PR TITLE
Putting Hair On Them Sharks 'And Lizards'

### PR DESCRIPTION
**Pull request by ArbyTheDragon**
A hopefully not in vain attempt at giving resident sharks and lizards the hairstyles they aught to have.

### CODE REVIEW

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@
 /datum/sprite_accessory/hair
 
 	icon = 'icons/mob/Human_face.dmi'	  // default icon for all hairs
+species_allowed = list("Human","Unathi","Akula")
```
Comment (ShadowLarkens) [inline code/modules/mob/new_player/sprite_accessories.dm]: this tabbing is broken

Comment (ArbyTheDragon) [inline code/modules/mob/new_player/sprite_accessories.dm]: Oops. Probably. Well, there was an attempt.

Comment (ArbyTheDragon) [inline code/modules/mob/new_player/sprite_accessories.dm]: It IS broken. And I fixed it. I just need to edit my commit.

**Comment by ArbyTheDragon** [comment]
It's fixed, I wrote it to a testing build on my computer, and **it works**.

**Pull request merged**